guest/internal/imports/imports.go

```diff
@@ -29,5 +29,8 @@ func k8sApiNodeName(ptr uint32, limit mem.BufLimit) (len uint32)
 //go:wasmimport k8s.io/api pod
 func k8sApiPod(ptr uint32, limit mem.BufLimit) (len uint32)
 
+//go:wasmimport k8s.io/api nodeToStatusMap
```

Comment:
It should be exported from k8s.io/scheduler since it's a scheduler specific type.

Reply:
Thanks. Fixed